ush/config.sh.RRFS_dev2

```diff
@@ -0,0 +1,103 @@
+MACHINE="jet"
```

Comment:
In the future, RRFS_dev1 and RRFS_dev2 (and other devs also) should only be different from config.sh.RRFS_dev*.
We need to talk if we have branch for other devs?

Reply:
I'm not sure what you mean here. The RRFS_dev1 branch should only have RRFS_dev1 configs. The other RRFS_dev* branches hold the config.sh* files specified for those branch configurations.

Reply:
The only difference between dev1 and dev2 is config.sh. I don't think it worth to maintain a branch for dev2. Others devs may be the same. Only one branch for RRFS and this RRFS has multiple config.sh. Like current dev1 has config.sh for conus and Alaska.

Reply:
We can discuss how to manage the other develop branches, but this branch should have config.sh.RRFS_dev1 in it, and this file is named config.sh.RRFS_dev2. It needs to be merged with config.sh.RRFS_dev1 so that all the contents reflect that this is dev1.

Reply:
Ok, got it. renamed config.sh.RRFS_dev2 as config.sh.RRFS_dev1

Reply:
This file should not exist in this branch.

Reply:
There are no dev2 branch now. The dev2 branch is holding the old system waiting to be merged into dev1. After merging all components from dev2. The dev2 branch doesn't need to be there. The dev2 and dev1 real-time run are based on the same branch with different configuration files. I think this dev2 configuration need to be in dev1 now.

Reply:
We can tag the current dev2 branch to ensure that we have it for reference moving forward, but we really should only be keeping the configuration files for dev1 in the dev1 branch.

Reply:
@hu5970 This file should not be included in the RRFS_dev1 branch.